sklego/meta/grouped_estimator.py

```diff
@@ -107,7 +79,7 @@ def __set_shrinkage_function(self):
             self.shrinkage_function_ = self.shrinkage
```

Comment:
Since we're no longer testing `GroupedEstimator`, might it be preferable if this implementation just points to `GroupedPredictor`? If there's any code improvements to be made, they'll still go to one place that way.

Reply:
That was my first thought, but I did it like this to prevent things from breaking at the users' end, because in the new version the argument `value_columns` does not exist anymore (instead all non-grouping columns are used, which I believe is more in line with the sklearn API). 

We could also rely on version management on the side of the user of course.

Then I would do:
```
from deprecated import deprecated
from .grouped_predictor import GroupedPredictor

@deprecated(
    version="0.5.2",
    reason="Please use `sklego.meta.GroupedPredictor` instead. "
    "This object will be removed from the meta submodule in version 0.7.0.",
)
def GroupedEstimator(*args, **kwargs):
    return GroupedPredictor(*args, **kwargs)

```

Would you prefer this?

Reply:
Now that I think of it, I prefer the previous method. `*args`/`**kwargs` don't play nice with the scikit-learn internals.